src/elm/Page/Community.elm

```diff
@@ -293,7 +273,8 @@ viewObjective loggedIn model metadata index objective =
     else
         div [ class "my-2" ]
             [ div
-                [ objDivStyle
+                [ class "px-3 py-4 bg-indigo-500 flex flex-col sm:flex-row sm:items-center cursor-pointer"
+                , classList [ ( "sm:h-auto", isOpen && objectiveDescriptionCharCount > 40 ), ( "sm:h-10", not isOpen || objectiveDescriptionCharCount < 40 ) ]
```

Comment:
Hmm, maybe we should use padding instead of a set height for this (or at least adjust the padding so it's consistent when it's open and when it's closed). It's very difficult to detect whether the text should wrap or not based on the length of the text (every character has a different width), so some objectives might be 40 characters long, but they might not wrap. Like this:
![2021-08-04-193131_1269x546_scrot](https://user-images.githubusercontent.com/39280468/128263793-d7e56e51-00e1-4390-adc6-608b85b8f68a.png)

Notice how the open objective gets taller than the others, but it doesn't wrap. Also, when you open something that does wrap, it's still kind of weird that the text goes down a bit.

Here's one closed:
![2021-08-04-193238_1212x98_scrot](https://user-images.githubusercontent.com/39280468/128263901-be2be155-fd0f-45cf-85fb-3a7a47e62ee9.png)

And the same one, but open:
![2021-08-04-193248_1214x451_scrot](https://user-images.githubusercontent.com/39280468/128263927-fd798bcd-b4cc-427a-a25e-43d30c42a30f.png)

Notice how when it's open, there's a bigger gap between the text and the top of the div.